docs/docker-hub/accounts.md

```diff
@@ -79,7 +79,7 @@ repository access rights to groups:
 - `Write` access users are able to push to non-automated repositories on the Docker
   Hub.
 - `Admin` access allows the user to modify the repositories "Description", "Collaborators" rights,
-  "Mark as unlisted", "Public/Private" status and "Delete".
+  "Mark as unlisted", "Public/Private" status and "Delete", and to retrieve build logs.
```

Comment:
Thanks for the clarification. This section on rights has some additional structural issues. If you don't mind, let's clarify them all.  If you do mind, happy to carry.

----> https://gist.github.com/moxiegirl/1c34957b9edc0f6d528c

### Repository group permissions

Use organization groups to manage the users that can interact with your repositories.

You must be in an organization's `Owners` group to create a new group, Hub
repository, or automated build. As an `Owner`, you then delegate the following
repository access rights to groups:

| Access Right | Description                                                                                                                                                                                                                                                                |
|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `Read`       | Users with this right can view, search, and pull a private repository.                                                                                                                                                                                                     |
| `Write`      | Users with this right can push to non-automated repositories on the Docker Hub.                                                                                                                                                                                            |
| `Admin`      | Users with this right can modify a repository's "Description", "Collaborators" rights. They can also mark a repository as unlisted, change its  "Public/Private" status and "Delete" the repository. Finally, `Admin` rights are required to read the build log on a repo. |
|              |                                                                                                                                                                                                                                                                            |

Regardless of their actual access rights, users with unverified email addresses
have `Read` access to the repository. Once they have verified their address,
they have their full access rights as granted on the organization.